src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.FruitStorage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
We don't test model classes since they don't have any logic except getters/setters which are never tested too